src/tests/ftest/cat_recovery/ddb.py

```diff
@@ -0,0 +1,666 @@
+
+        Inserted objects: self.ioreqs
+        Inserted dkeys: self.dkeys
+        Inserted akeys: self.akeys
+        Inserted data: self.data_list
+
+        Args:
+            object_count (int): Number of objects to insert.
+            dkey_count (int): Number of dkeys to insert.
+            akey_count (int): Number of akeys to insert.
+        """
+        self.container.open()
+
+        for obj_index in range(object_count):
+            # Insert object.
+            self.ioreqs.append(IORequest(
+                context=self.context, container=self.container.container, obj=None,
+                objtype=DaosObjClass.OC_S1))
+
+            for dkey_index in range(dkey_count):
+                # Prepare the dkey to insert into the object.
+                dkey_str = " ".join(
+                    [SAMPLE_DKEY, str(obj_index), str(dkey_index)]).encode("utf-8")
+                self.dkeys.append(
+                    create_string_buffer(value=dkey_str, size=len(dkey_str)))
+
+                for akey_index in range(akey_count):
+                    # Prepare the akey to insert into the dkey.
+                    akey_str = " ".join(
+                        [SAMPLE_AKEY, str(obj_index), str(dkey_index),
+                         str(akey_index)]).encode("utf-8")
+                    self.akeys.append(
+                        create_string_buffer(value=akey_str, size=len(akey_str)))
+
+                    # Prepare the data to insert into the akey.
+                    data_str = " ".join(
+                        [SAMPLE_DATA, str(obj_index), str(dkey_index),
+                         str(akey_index)]).encode("utf-8")
+                    self.data_list.append(
+                        create_string_buffer(value=data_str, size=len(data_str)))
+                    c_size = ctypes.c_size_t(ctypes.sizeof(self.data_list[-1]))
+
+                    # Insert dkeys, akeys, and the data.
+                    self.ioreqs[-1].single_insert(
+                        dkey=self.dkeys[-1], akey=self.akeys[-1],
+                        value=self.data_list[-1], size=c_size)
+
+    def get_vos_file_path(self):
+        """Get the VOS file path.
+
+        Returns:
+            str: VOS file path such as /mnt/daos/<pool_uuid>/vos-0
+
+        """
+        hosts = NodeSet(self.hostlist_servers[0])
+        vos_path = os.path.join("/mnt/daos", self.pool.uuid.lower())
+        command = " ".join(["sudo", "ls", vos_path])
+        cmd_out = run_pcmd(hosts=hosts, command=command)
+
+        vos_file = None
+        for file in cmd_out[0]["stdout"]:
+            # Assume the VOS file has "vos" in the file name.
+            if "vos" in file:
+                vos_file = file
+                break
+
+        if not vos_file:
+            self.fail(
+                "vos file wasn't found in /mnt/daos/{}".format(self.pool.uuid.lower()))
+        else:
+            self.log.info("vos_file: %s", vos_file)
+
+        return vos_file
```

Comment:
Recommend simplifying
```suggestion
        for file in cmd_out[0]["stdout"]:
            # Assume the VOS file has "vos" in the file name.
            if "vos" in file:
                self.log.info("vos_file: %s", file)
                return file

        self.fail(
            "vos file wasn't found in /mnt/daos/{}".format(self.pool.uuid.lower()))
            
        return None  # to appease pylint
```

Reply:
Done. Thanks